### PR TITLE
Replace outdated ES6 page with node.green

### PR DESCRIPTION
**Pull request by fhemberger**
This should land one v6 comes out.
Ref:
- https://github.com/nodejs/nodejs.org/issues/671
- https://github.com/nodejs/nodejs.org/pull/674

### CODE REVIEW

File: setup/www/resources/config/nodejs.org [hunk 1]
```diff
@@ -392,4 +392,5 @@ server {
     rewrite ^/images/(.*)                                         https://$server_name/static/images/$1 permanent;
 
     rewrite ^/windows-environment$                                https://github.com/nodejs/node/wiki/Windows-Environment permanent;
+    rewrite ^/en/docs/es6/                                        http://node.green permanent;
```
Comment (evanlucas) [inline setup/www/resources/config/nodejs.org]: would be nice if node.green supported https. /cc @williamkapke

**Comment by ChALkeR** [comment]
Does node.green explain the «shipping»/«staged»/«in progress» status of the features? Afaik it doesn't, and that information is valuable there.

**Comment by fhemberger** [comment]
@ChALkeR Please keep the discussion in the main thread: https://github.com/nodejs/nodejs.org/issues/671

**Comment by fhemberger** [comment]
Closing this PR, we're keeping this page for now and just link to node.green

**Pull request closed without merging**